app/views/layouts/application.html.erb

```diff
@@ -2,13 +2,69 @@
 <html>
 <head>
   <title>Betsy</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
```

Comment:
There is a bootstrap gem for rails you can use instead of linking to it externally here.